ecs.tf

```diff
@@ -86,11 +86,19 @@ locals {
 
   # merge and flatten decisions
   container_defs = flatten(concat(local.app_container,  local.firelens_dd_app_container))
+
+  # use cloudwatch container insights if containerInsights is enabled
+  container_insights_setting = {
+    name  = "containerInsights"
+    value = var.containerInsights
```

Comment:
This value should be "enabled" or "disabled" from looking at the docs, maybe a little ternary here is in order?

```
var.containerInsights ? "enabled" : "disabled"
```

Reply:
good spot - I'll change it